hack/unit.sh

```diff
@@ -2,5 +2,17 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+WMCO_ROOT=$(dirname "${BASH_SOURCE}")/..
+cd "${WMCO_ROOT}"
+
+# Account for environments where GOFLAGS is not set by setting goflags to an empty string if GOFLAGS is not set
+goflags=${GOFLAGS:-}
+
+# The golang 1.13 image used in CI enforces vendoring. Workaround that by unsetting it.
+if [[ "$goflags" == *"-mod=vendor"* ]]; then
+  unset GOFLAGS
+fi
```

Comment:
There is no mention of this in the commit or PR message. Please call that out.

Reply:
addressed

Reply:
@sebsoto @ravisantoshgudimetla I would like some feedback regarding this. Should this go into a separate commit?

Reply:
A separate commit is good.